tests/serac_stdfunction_coefficient.cpp

```diff
@@ -194,3 +194,69 @@ TEST_F(StdFunctionCoefficientTest, EssentialBC)
     }
   }
 }
+
+/**
+   Create the ess_vdof_list for all dofs that are x = 0
+ */
+TEST_F(StdFunctionCoefficientTest, EssentialBCCube)
+{
+  // Create an indicator function to set vertex at the origin
+  StdFunctionVectorCoefficient origin_bc(3, [](Vector &x, Vector &X) {
```

Comment:
The other ones use `pfes_v->GetVDim()` as the first argument. What is this test doing?

Reply:
@jonwong12 can answer since he's the one who wrote it, but I believe it's setting the appropriate dofs for a component-wise boundary condition.

Reply:
The `3` is the vector dimension for the vector coefficient. I believe it is passed directly to the underlying base constructor for `VectorCoefficient`. I think `mesh->SpaceDim()` might actually be more appropriate.